server/views/pages/visit/summary.njk

```diff
@@ -67,22 +68,63 @@
         {% set cancelButtonClasses = 'govuk-!-margin-top-5' %}
         {% if updateJourneyEnabled %}
           {% set cancelButtonClasses = 'govuk-!-margin-top-5 govuk-button--secondary' %}
+
```

Comment:
Many if else blocks due to updateJourney feature flag being used. This can be made to be more concise and clear once we no longer require this flag